docs/dev/cli-commands.md

```diff
@@ -0,0 +1,38 @@
+## CLI Basics
+
+The main tasks of the CLI component are:
+- parse and call the commands
+- call the method to transform a given topology to a custom platform (start / stop / transform platform)
+- show current status of transformation/ deployment
+- print detailed logs if wanted
+- show supported platforms
+- create threads to provide asynchronous operations
+
+## Supported Commands
+
+Each of the following commands except help have these options: `-d` or `--debug`, `-u` or `--usage`, `-v` or `--verbose`.
+Parameters of the commands are represented as `<name>`.
+
+Options explanation:
+- Debug: enable/ disable Debugging
+- Usage: Explain the usage of the specific command
+- Verbose: enable/ disable more detailed Information
+
+| Command | Description | additional Options |
+| ------- | ----------- | ------------------ |
+| add | uploads a specific CSAR Archive to the Transformator, Usage: `add <CSARname>`| |
+| delete | deletes specific CSAR Archives, Transformations or all of them, Usage: `delete <CSARname>`, `delete <Transformationname>` or `delete -c`/ `delete -t` / `delete -c -t` | `-c` or `--csar`, `-t` or `--transformation` |
```

Comment:
What effect do the flags have? What arguments do I have to pass with them? After reading the table I cant tell that.